.github/workflows/main.yml

```diff
@@ -357,24 +357,24 @@ jobs:
       shell: bash
     - name: Check Docfx cache
       id: check-docfx-cache
-      if: ${{ contains(github.ref, 'release') }}
+      if: ${{ contains(github.head.ref, 'release') }}
```

Comment:
```suggestion
      if: ${{ isRelease }}
```

Reply:
Now I understand what you mean. You can't do it that way. `isRelease` is a var that exist only for ytt. That has no meaning in the outputted yaml file. It's only living in the template accepted by ytt.

Reply:
I knew it will be something like that. Thanks for clarifying! Those CI configs are still quite confusing when you're not used to working with them. 🙈

Reply:
No problem. Yes yes, we know it looks a bit weird at first. We all felt that way at the beginning 😛